### PR TITLE
Add Firefox built for you landing page [fix #14985]

### LINKED ISSUE #14985: EU Firefox | Built for You | Landing Page
Hello. Please refer to **[this](https://docs.google.com/document/d/15LZeYNFYEnhaM00P-1UMTs1lQ9TWj7kmDkEv4Ub3tg8/edit)** brief document for details. Thank you.

### PR DESCRIPTION
**Pull request by craigcook**
## One-line summary
Adds "Built For You" campaign landing page in FR and DE, each with five variations (just swapping the header graphic; they'll get direct links from ads, no Traffic Cop experiment).

## Issue / Bugzilla link
#14985 

## Testing
http://localhost:8000/de/firefox/built-for-you/?v=1 - Pinocchio video
http://localhost:8000/de/firefox/built-for-you/?v=2 - Superhero video
http://localhost:8000/de/firefox/built-for-you/?v=3 - Sloth
http://localhost:8000/de/firefox/built-for-you/?v=4 - Alien
http://localhost:8000/de/firefox/built-for-you/?v=5 - Boring normal human (also the default with no param)

(French is same as above, but /fr/)

### CODE REVIEW

**Comment by craigcook** [comment]
Marking Do Not Merge for now because I'm waiting for recropped assets in a 4:3 aspect ratio for variants 1 and 2.

File: media/css/firefox/built-for-you/landing.scss [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+    background-color: $color-black;
+
+    &:hover,
+    &:focus,
+    &:active {
+        background-color: $color-white;
+        color: $color-black;
+        border-color: $color-black;
+    }
+}
```
Comment (alexgibson) [inline media/css/firefox/built-for-you/landing.scss:38]: **issue:** (non-blocking) I know we're about to hit a redesign, but changing the buttons to black for this page does feel kinda inconsistent with our regular pages?

**Review by alexgibson — APPROVED** [review]
Well coded, and a very clear and simple set of landing pages. Nice work!

Do the EU team want analytics on this page? If so, adding the URL to the allow-list will enable the consent banner to show: https://github.com/mozilla/bedrock/blob/main/media/js/base/consent/allow-list.es6.js

File: bedrock/firefox/templates/firefox/built-for-you/includes/v1-pinocchio-de.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+<div class="c-hero-video">
+  <video preload="auto" playsinline autoplay muted loop poster="{{ static('img/firefox/built-for-you/pinocchio-1000.jpg') }}" width="800" height="800">
+    <source src="https://assets.mozilla.net/video/built-for-you/video-pinocchio-de.webm" type="video/webm">
+  </video>
+
+  {{ resp_img('img/firefox/built-for-you/pinocchio-800.jpg',
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/built-for-you/includes/v1-pinocchio-de.html:12]: **Nit:** (non-blocking) adding some some `sizes` info to these image might help browsers to ppick out the most appropriate size a bit better.

File: media/css/firefox/built-for-you/landing.scss (hunk 1 above)
Comment (alexgibson) [reply media/css/firefox/built-for-you/landing.scss:38]: There's also a blue CTA in the nav still (not not a blocker):

image

Comment (craigcook) [reply media/css/firefox/built-for-you/landing.scss:38]: > **issue:** (non-blocking) I know we're about to hit a redesign, but changing the buttons to black for this page does feel kinda inconsistent with our regular pages?

I felt the same, but it's what was designed and approved so clearly intentional 🤷

**Review by alexgibson — APPROVED** [review]
r++

**Pull request merged**